pkg/commands/oscommands/os_default_platform.go

```diff
@@ -4,14 +4,23 @@
 package oscommands
 
 import (
+	"os"
 	"runtime"
 )
 
+func getShell() string {
+	defaultShell := "bash"
+	if shell := os.Getenv("SHELL"); shell != defaultShell {
```

Comment:
This logic isn't quite right:
If shell != default shell, we return shell
if shell == default shell, we return default shell which is really just shell

So we're returning shell either way.

I would instead say that if SHELL is blank string, we default to bash

Reply:
~We presume the shell is `bash`.~

~If we read the environment variable `SHELL` (which actually returns the absolute path, my bad) and it turns out to be `zsh` (`!= bash`), we return `zsh` which... yeah, okay~

> So we're returning shell either way.

Well, when you put it _that_ way... :sweat_smile: